src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2115,42 +2219,60 @@
           short-summary: Port number on which to receive traffic.
         - name: --dest-address
           short-summary: The IP address or URI at which to receive traffic.
+    examples:
+        - name: Checks connectivity between two virtual machines in the same resource group over port 80.
+          text: az network watcher test-connectivity -g MyResourceGroup --source-resource MyVmName1 --dest-resource MyVmName2 --dest-port 80
+        - name: Checks connectivity between two virtual machines in the same subscription in two different resource groups over port 80.
+          text: |
+            Vm1Id=$(az vm show --resource-group MyResourceGroup--name MyVm1 --query id --out tsv) \\
+                \\n\\n Vm2Id=$(az vm show --resource-group MyResourceGroup--name MyVm2 --query id --out tsv) \\
+                \\n\\n az network watcher test-connectivity --source-resource $MyVmId1 --dest-resource $MyVmId2 --dest-port 80
 """
 
 helps['network watcher show-next-hop'] = """
     type: command
     short-summary: Get information on the 'next hop' for a VM.
-    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.
+    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located. For more information about show-next-hop visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-check-next-hop-cli
+    examples:
+        - name: Get the next hop from a VM's assigned IP address to a destination at 10.1.0.4.
+          text: az network watcher show-next-hop -g MyResourceGroup --vm MyVm --source-ip 10.0.0.4 --dest-ip 10.1.0.4
 """
 
 helps['network watcher show-security-group-view'] = """
     type: command
     short-summary: Get detailed security information on a VM for the currently configured network security group.
+    long-summary: For more information about security-group-view visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-security-group-view-cli
+    examples:
+        - name: Get the network security group information for the specified VM.
+          text: az network watcher show-security-group-view -g MyResourceGroup --vm MyVm
 """
 
 helps['network watcher show-topology'] = """
     type: command
     short-summary: Get the network topology of a resource group.
+    long-summary: For more information about show-topology visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-topology-cli
     parameters:
         - name: --resource-group -g
           short-summary: The name of the target resource group to perform topology on.
         - name: --location -l
           short-summary: Location. Defaults to the location of the target resource group.
-          long-summary: Topology information is only shown for resources within the target
+          long-summary: >
+            Topology information is only shown for resources within the target
```

Comment:
`long-summary` values should end with a `.`. There may be other places in the PR where this applies.